pkg/core/capability/manual/manual.go

```diff
@@ -0,0 +1,85 @@
+package manual
+
+import (
+	"errors"
+	"reflect"
+	"soarca/internal/logger"
+	"soarca/pkg/core/capability"
+	"soarca/pkg/interaction"
+	"soarca/pkg/models/cacao"
+	"soarca/pkg/models/execution"
+	"time"
+)
+
+var (
+	component = reflect.TypeOf(ManualCapability{}).PkgPath()
+	log       *logger.Log
+)
+
+const (
+	manualResultVariableName = "__soarca_manual_result__"
+	manualCapabilityName     = "soarca-manual-http"
+	fallbackTimeout          = time.Minute * 1
+)
+
+func New(controller interaction.ICapabilityInteraction,
+	channel chan interaction.InteractionResponse) ManualCapability {
+	// channel := make(chan interaction.InteractionResponse)
+	return ManualCapability{interaction: controller, channel: channel}
```

Comment:
Can we discuss on whether we need one channel per whole playbook execution, or one channel per each manual command execution?

As you implemented it, is one per playbook execution, but I was thinking we might need one per command execution.

This is because if we have a parallel step with two manual commands, assuming parallel steps is actually asynchronous, then if we have only one manual capability per playbook execution, we might get the wrong response in a parallel branch, if the response from the other parallel branch was executed first, right?